camera_param:
  mtx:
    mtx0: 1149.6
    mtx1: 0.0
    # mtx2: 944.2056
    mtx2: 937.2056
    mtx3: 0.0
    mtx4: 1149.9
    mtx5: 559.5819
    mtx6: 0.0
    mtx7: 0.0
    mtx8: 1.0
  dist:
    dist0: 0.0
    dist1: 0.0
    dist2: 0.0
    dist3: 0.0
    dist4: 0.0
  height: 488
  resolution:
    resolution0: 1920
    resolution1: 1080

  # 相机相对云台转轴 向右为x
  transform:
    transform0: 0  # x
    transform1: 152  # y
    angle: 0  # 逆时针为正
show: 1

head_param:
  # 云台相对底盘中心 向右为x
  transform:
    transform0: 0  # x
    transform1: 261.3  # y